testcafe/tests/pages/drive/drive-model.js

```diff
@@ -10,14 +10,24 @@ export default class DrivePage {
   async waitForLoading({ isNotAvailable, isFull } = {}) {
     await t
       .expect(selectors.contentPlaceHolder.exists)
-      .notOk('Content placeholder still displayed')
+      .notOk(
+        'waitForLoading - Page didnt Load : selectors.contentPlaceHolder still displayed'
```

Comment:
I think `waitForLoading` is not helpful here. On the other hand saying what page didn't load would probably help with troubleshooting.

Reply:
Same goes for all the other instances of `waitForLoading`.

Reply:
Done